rnd/autogpt_server/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM python:3.11-slim-buster as server_base
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+
+WORKDIR /app
+
+RUN apt-get update \
+    && apt-get install -y build-essential curl ffmpeg wget libcurl4-gnutls-dev libexpat1-dev gettext libz-dev libssl-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && wget https://github.com/git/git/archive/v2.28.0.tar.gz -O git.tar.gz \
+    && tar -zxf git.tar.gz \
+    && cd git-* \
+    && make prefix=/usr all \
+    && make prefix=/usr install
```

Comment:
Why not simply `apt-get install git`?

Reply:
I was getting a lot of errors and then I pulled this from codex dockerfile and it works

Reply:
Strange, works fine here: https://github.com/Significant-Gravitas/AutoGPT/blob/aa8ca37f86c21370a71d3b317285df166304f9c0/Dockerfile.autogpt#L13-L14